charts/cloudserver-front/values.yaml

```diff
@@ -29,6 +29,13 @@ image:
   tag: 0.2.1
   pullPolicy: IfNotPresent
 
+proxy:
+  # If you want to use an http proxy, add the respective endpoints after
+  # 'http:' and/or 'https:'. If the http proxy endpoint is set but the https
+  # one isn't, the http proxy will be used for https traffic anyway.
```

Comment:
Change `http` into `HTTP`, `https` to `HTTPS` (in the comment, not the actual variable names, nor references to the variable names in the comment, of course), and change `anyway` into `as well`, and I'm all happy and such.